src/editors/data/redux/thunkActions/requests.test.js

```diff
@@ -1,15 +1,26 @@
-import { actions } from '..';
 import { RequestKeys } from '../../constants/requests';
 import api from '../../services/cms/api';
 import * as requests from './requests';
+import { actions, selectors } from '../index';
 
-jest.mock('data/services/lms/api', () => ({
-  initializeApp: (locationId) => ({ initializeApp: locationId }),
-  fetchSubmissionStatus: (submissionUUID) => ({ fetchSubmissionStatus: submissionUUID }),
-  fetchSubmission: (submissionUUID) => ({ fetchSubmission: submissionUUID }),
-  lockSubmission: ({ submissionUUID }) => ({ lockSubmission: { submissionUUID } }),
-  unlockSubmission: ({ submissionUUID }) => ({ unlockSubmission: { submissionUUID } }),
-  updateGrade: (submissionUUID, gradeData) => ({ updateGrade: { submissionUUID, gradeData } }),
+const studioEndpointUrl = 'ASVouAGE.S$e';
+const blockId = 'CHalockIDmaiw@alcioUSness';
+const blockType = 'hTMl';
+const courseId = 'BEnX:INtrOToUNIttEsTing';
+const title = 'MYbLock';
+
+jest.mock('../app/selectors', () => ({
+  studioEndpointUrl: () => (studioEndpointUrl),
```

Comment:
```
myTestSelector: (state) => ({ myTestSelector: state }),
myComplicatedSelector: (state, props) => ({ myComplicatedSelector: { state, props } }),
```

Reply:
I don't get this, because the selectors we are mocking don't output the whole state, and why would we mock something with different than desired behavior?

Reply:
Do you mean something like? `myTestSelector: (state) => ({ state[myTestSelector]})` because that would be closer to what a selector is, but I'm not sure I like that either?

Reply:
We discussed this offline, but in case anyone comes here looking for the answer later:
We are mocking the selectors here to give a return value that includes an identification of which selector was called, and how it was called, in order to avoid having to test those pieces independently, and rather than simply checking that we get the output of a selector when called with *any* arguments.

It is much more important in **unit tests** to verify what the code is specifically trying to do, rather than to perfectly model the use case it is intended to serve.